tests/preprocessing/test_positionfixes.py

```diff
@@ -110,15 +121,15 @@ def test_duplicate_pfs_warning(self, example_positionfixes):
         pfs_duplicate_all = pfs_duplicate_loc.copy()
         pfs_duplicate_all.loc[0, "tracked_at"] = pfs_duplicate_all.loc[1, "tracked_at"]
 
+        warn_string = ".* duplicates were dropped from your positionfixes."
         with pytest.warns(None) as record:
```

Comment:
Just a general questions for understanding: pytest.warns(None) means that all warnings are catched, while pytest.warns(UserWarning) means that you only check for warnings that were created in this package?

Reply:
Yes. pytest.warns(None) captures warnings of all types, see [here](https://docs.pytest.org/en/6.2.x/warnings.html#recwarn). pytest.warns(UserWarning) only captures the warning type `UserWarning`. I think when raising a warning, we can define the type of the warning, and `UserWarning` is the default type if not specified, see [here](https://docs.python.org/3/library/warnings.html#warnings.warn)